From: Iain Lane <laney@debian.org>
Date: Mon, 17 Feb 2020 13:40:37 +0100
Subject: time-zone-path test: Update for what our system ICU returns

  1) I'm not sure what "/this-part-is-ignored/" is meant to mean, looks
     like a path to the zoneinfo file?
  2) When handled by ICU, these return offsets and not timezone names.

Upstream issue: https://phabricator.services.mozilla.com/D5827

Forwarded: no (but should be)
---
 js/src/tests/non262/Date/time-zone-path.js | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/js/src/tests/non262/Date/time-zone-path.js b/js/src/tests/non262/Date/time-zone-path.js
index 3422022..8773e44 100644
--- a/js/src/tests/non262/Date/time-zone-path.js
+++ b/js/src/tests/non262/Date/time-zone-path.js
@@ -17,11 +17,11 @@ assertEq(timeZoneName(), "Eastern European Summer Time");
 setTimeZone("::Europe/London"); // two colons, invalid
 assertEq(timeZoneName(), "Coordinated Universal Time");
 
-setTimeZone("/this-part-is-ignored/zoneinfo/America/Chicago");
-assertEq(timeZoneName(), "Central Daylight Time");
+setTimeZone("/usr/share/zoneinfo/America/Chicago");
+assertEq(timeZoneName(), "GMT-06:00");
 
-setTimeZone(":/this-part-is-ignored/zoneinfo/America/Phoenix");
-assertEq(timeZoneName(), "Mountain Standard Time");
+setTimeZone(":/usr/share/zoneinfo/America/Phoenix");
+assertEq(timeZoneName(), "GMT-07:00");
 
 setTimeZone("::/this-part-is-ignored/zoneinfo/America/Los_Angeles"); // two colons, invalid
 assertEq(timeZoneName(), "Coordinated Universal Time");

